Manual Brake - Further update (#280)
openrails/openrails: Source/Documentation/Manual/physics.rst

To control the application and release rates on the brake use the ``EngineBrakesControllerMaxApplicationRate`` and 
``EngineBrakesControllerMaxReleaseRate`` parameters.

> A steam effect can be added by adding the following special effects to the wagon ``SteamBrakeFX``. This special effect will 
> turn on and off with the brake operation and can be used to model steam leakage of the steam brake cylinder, etc.

Reviewer: ```suggestion
The ``SteamBrakeFX`` special effect, if added to the wagon, will turn on and 
off with the brake operation and can be used to model steam leakage of the 
steam brake cylinder, etc.
```
I suggest rewording this to flow more naturally.

Author: Suggested change added.
